Policies/github-terms-of-service.md

```diff
@@ -36,95 +36,118 @@ Thank you for using GitHub! We're happy you're here. Please read this Terms of S
+Your use of the Website and Service must not violate any applicable laws (including copyright or trademark laws, export control or sanctions laws, or other laws in your jurisdiction), and you are responsible for this compliance.
 
-You agree that you will not under any circumstances violate our [Acceptable Use Policies](https://help.github.com/articles/github-acceptable-use-policies) or [Community Guidelines](https://help.github.com/en/articles/github-community-guidelines).
+You will not under any circumstances violate our [Acceptable Use Policies](https://help.github.com/articles/github-acceptable-use-policies) or [Community Guidelines](https://help.github.com/en/articles/github-community-guidelines).
 
 ### D. User-Generated Content
 **Short version:** *You own content you create, but you allow us certain rights to it, so that we can display and share the content you post. You still have control over your content, and responsibility for it, and the rights you grant us are limited to those we need to provide the service. We have the right to remove content or close Accounts if we need to.*
 
 #### 1. Responsibility for User-Generated Content
-You may create or upload User-Generated Content while using the Service. You are solely responsible for the content of, and for any harm resulting from, any User-Generated Content that you post, upload, link to or otherwise make available via the Service, regardless of the form of that Content. We are not responsible for any public display or misuse of your User-Generated Content.
+You are solely responsible for the content of, and for any harm resulting from, any User-Generated Content that you post, upload, link to or otherwise make available via the Service, regardless of the form of that Content. We are not responsible for any public display or misuse of your User-Generated Content.
 
 #### 2. GitHub May Remove Content
 We do not pre-screen User-Generated Content, but we have the right (though not the obligation) to refuse or remove any User-Generated Content that, in our sole discretion, violates any GitHub terms or policies.
 
 #### 3. Ownership of Content, Right to Post, and License Grants
-You retain ownership of and responsibility for Your Content. If you're posting anything you did not create yourself or do not own the rights to, you agree that you are responsible for any Content you post; that you will only submit Content that you have the right to post; and that you will fully comply with any third party licenses relating to Content you post.
+You:
+1) retain ownership of, and responsibility for, Your Content, 
+2) will only submit Content that you have the right to post,
+3) are responsible for such submitted Content (even when you don't own the Content), AND
+4) will fully comply with any third party licenses relating to such Content.
```

Comment:
`must`

Reply:
🆗  Happy to change 'will only' and 'will fully' to 'must only' and 'must fully' respectively.

However, is it necessary? Can anyone with legal expertise comment? The previous text used 'will' and not 'must'...

Reply:
🆗 Last commit in patch now implements suggestion.